src/protagonist/DLCS.AWS/SNS/ITopicPublisher.cs

```diff
@@ -10,11 +10,11 @@ public interface ITopicPublisher
     /// <param name="messages">A collection of notifications to send</param>
     /// <param name="cancellationToken">Current cancellation token</param>
     /// <returns>Boolean representing the overall success/failure status of all requests</returns>
-    public Task<bool> PublishToAssetModifiedTopic(IReadOnlyList<AssetModifiedNotification> messages,
+    public Task<bool> PublishToAssetModifiedTopic(IReadOnlyList<AssetModifiedNotification> messages, 
         CancellationToken cancellationToken);
 }
 
 /// <summary>
 /// Represents the contents + type of change for Asset modified notification
 /// </summary>
-public record AssetModifiedNotification(string MessageContents, ChangeType ChangeType);
\ No newline at end of file
+public record AssetModifiedNotification(string MessageContents, ChangeType ChangeType, bool EngineNotified);
```

Comment:
Is it wrong to have an "Update"-specific property for what should be a generic representation of an asset modified notification? If we needed to add other attributes would these be chained here? (e.g. if we wanted to have the `DeleteFrom` as an attribute, would that be added here despite only being relevant for deletes?).

Is there a way to keep `AssetModifiedNotification` as a generic carrier of notification data without having ChangeType specific props? (similar comment to one for `AssetModificationRecord`)